### PR TITLE
Add Azure MySQL

### PR DESCRIPTION
**Pull request by spacentropy**
Hello! Thanks for the awesome tool.
Here's the Azure MySQL compatibility.

### CODE REVIEW

**Comment by charles-001** [comment]
Hi @spacentropy - thank you for your contribution! I just have 1 comment about it :)

**Comment by charles-001** [comment]
@spacentropy Following up on this - did you see my previous comment?

**Comment by spacentropy** [comment]
@charles-001 sorry I don't see any comments. Maybe you've left a comment but did not finish the review?

**Review by charles-001 — CHANGES_REQUESTED** [review]

**Comment by charles-001** [comment]
> @charles-001 sorry I don't see any comments. Maybe you've left a comment but did not finish the review?

Ah... I left a comment, but it seems I needed to request a change. Sorry about that!

File: dolphie/__init__.py [hunk 1]
```diff
@@ -176,12 +177,17 @@ def db_connect(self):
...
+            self.host_distro = "Azure MySQL"
+            self.azure = True
         else:
             self.host_distro = "MySQL"
 
-        # For RDS, we will use the host specified to connect with since hostname isn't related to the endpoint
+        # For RDS and Azure, we will use the host specified to connect with since hostname isn't related to the endpoint
         if self.aws_rds:
             self.mysql_host = f"{self.host.split('.rds.amazonaws.com')[0]}:{self.port}"
+        if self.azure:
```
[the remark this answers is not in the document]
Comment (spacentropy) [reply dolphie/__init__.py]: Right, fixed

**Pull request merged**